.github/workflows/core.yml

```diff
@@ -326,12 +326,18 @@ jobs:
           key: ${{ runner.os }}-zeppelin-${{ hashFiles('**/pom.xml') }}
           restore-keys: |
             ${{ runner.os }}-zeppelin-
+      - name: Install R
+        shell: bash
+        run: |
+          whoami
+          sudo apt-get update -y
+          sudo apt-get install -y r-base r-cran-data.table r-cran-evaluate r-cran-base64enc r-cran-knitr r-cran-ggplot2 r-cran-irkernel r-cran-shiny r-cran-googlevis
```

Comment:
During the debug phase, I found that conda seems replies bash login mode to activate the environment, which may result in that YARN container can not see the conda-managed python/R packages, then the Spark YARN cluster mode test failed because Driver(AM) can not see conda-managed python/R packages.

Considering the raising of Docker technology, and currently, most of CI environments are containerized, I would like to use the OS-managed packages in CI environment to simplify the package management. (As long as that Apache Spark project installs dependencies in OS-level in CI)

Reply:
@Reamer @jongyoul My experiment shows that the APT-installed OS-level R packages work well on CI, but Conda-managed does not. I don't have much knowledge of R, do you think using OS-level R packages is OK? Or I should continually dig the root cause and fix the conda's issue?

Reply:
I am a bit confused. In the PR you write that Livy tests are not executed, which is due to the surefire or failsafe version.

Now the Spark-Yarn tests are not executed correctly due to the conda version. In my opinion, the Spark and Yarn tests work with the conda environment.

Originally Zeppelin relies on conda because we can install an OS-independent R, python environment (also in Docker images). If we now switch back to OS packages, it feels like a step backwards.

Reply:
I think I have managed to reproduce the behavior.

Reply:
@Reamer sorry for the late reply, just back to the office after a long public holiday.

> I think I have managed to reproduce the behavior.

Yes, it's easy to reproduce in CI if we only bump the Maven plugin version.

Reply:
I am now able to run the YARN cluster tests with the following changes.
https://github.com/Reamer/zeppelin/commit/ed1fbc4f21f0c972e239c8b31369ef7b38e5910e

Please note that I have made a lot of changes to the Zeppelin tests in this branch. Theoretically, the property changes in your branch should be sufficient for the tests to run.